components/concept/sync/src/main/java/mozilla/components/concept/sync/Sync.kt

```diff
@@ -60,14 +60,6 @@ interface LockableStore : SyncableStore {
  * Describes a "sync" entry point for a storage layer.
  */
 interface SyncableStore {
-    /**
-     * Performs a sync.
-     *
-     * @param authInfo Auth information necessary for syncing this store.
-     * @return [SyncStatus] A status object describing how sync went.
-     */
-    suspend fun sync(authInfo: SyncAuthInfo): SyncStatus
```

Comment:
nit: Should we remove this? or provide a implementation on the new class that return failure?